src/app/components/register/register.component.ts

```diff
@@ -10,16 +15,40 @@ import {CustomRegex} from '../../utils/CustomRegex';
 })
 export class RegisterComponent implements OnInit {
 
+  registerRequest = false;
+  errorMsg = '';
+
   registerFormGroup: FormGroup;
 
   constructor(formBuilder: FormBuilder,
+              private authService: AuthService,
+              private translate: TranslateService,
               private router: Router) {
     this.registerFormGroup = formBuilder.group({
+      userName: [null,
```

Comment:
Bitte mal in die Api Doku schauen (Swagger). Einige der Felder haben maxlength-Validatoren oder pattern-Validatoren, die sinnigerweise auch im Frontend verwendet werden sollten

Reply:
![grafik](https://user-images.githubusercontent.com/39371377/107411974-71a12600-6b0f-11eb-9d6a-e2c243eaa484.png)

Reply:
Hinweis: Das Pattern bei password wird falsch angezeigt, richtig wäre folgendes: 
![grafik](https://user-images.githubusercontent.com/39371377/107412212-b4fb9480-6b0f-11eb-8a07-f0f4264ebe17.png)